libbeat/processors/annotate/kubernetes/kubernetes.go

```diff
@@ -52,18 +52,44 @@ func newKubernetesAnnotator(cfg common.Config) (processors.Processor, error) {
 		return nil, err
 	}
 
+	//Load default indexer configs
+	if config.DefaultIndexers.Enabled == true {
+		for key, cfg := range Indexing.defaultIndexerConfigs {
+			config.Indexers = append(config.Indexers, map[string]common.Config{key: cfg})
```

Comment:
would this need the rwmutex part? processor init is anyway something that is serial